source/module/ActiveDirectoryFunctions.ps1

```diff
@@ -112,17 +113,30 @@ function New-SystemData
         "AllServers"    {$targetMachines = @(Get-ADComputer -Filter {OperatingSystem -like "**server*"} -Properties "operatingsystem", "distinguishedname") ; break }
         "Full"          {$targetMachines = @(Get-ADComputer -Filter * -Properties "operatingsystem", "distinguishedname") ; break }
         "Targeted"      {$targetMachines = @(Get-AdComputer -Identity "$ComputerName" -Properties "operatingsystem","distinguishedname") ; break }
-        "Local"
+        "Local" {
+
+        if ($osVersion -like '*Server*') {
+           
+            $targetMachines = @(
+                @{
+                    Name              = $env:computerName
+                    OperatingSystem   = $osVersion
+                    distinguishedname = "Servers"
+                }
+            )
+        }    
+        else 
         {
             $targetMachines = @(
                 @{
-                    Name = $env:computerName
-                    OperatingSystem     = "Windows 10"
-                    distinguishedname   = "Computers"
+                    Name              = $env:computerName
+                    OperatingSystem   = $osVersion
+                    distinguishedname = "Computers"
                 }
             )
         }
     }
+}
 
     Write-Output "`tIdentifying Organizational Units for $($targetMachines.count) systems."
```

Comment:
Move this line under line 143 so that it only says it's identifying the OUs if it's not running with the localhost switch